windows/src/desktop/kmconfig/Makefile

```diff
@@ -6,7 +6,7 @@
 
 build: version.res manifest.res #icons
     $(DELPHI_MSBUILD) kmconfig.dproj /p:Platform=Win32
-    $(SENTRYTOOL_DELPHIPREP) $(WIN32_TARGET_PATH)\kmconfig.exe -dpr kmconfig.exe
+    $(SENTRYTOOL_DELPHIPREP) $(WIN32_TARGET_PATH)\kmconfig.exe -dpr kmconfig.dpr
```

Comment:
Where does `sentrytool.dpr` get used?

Reply:
`sentrytool.dpr` is the source project for sentrytool. I'm not sure I am understanding your question though.

Reply:
I was just checking the parameter wasn't supposed to be `...\kmconfig.exe -dpr sentrytool.dpr`

Reply:
No, this is using sentrytool to process the target executable and uses the target's dpr for additional info when rewriting the executable.